core/src/com/unciv/logic/automation/unit/CityLocationTileRanker.kt

```diff
@@ -110,6 +110,9 @@ object CityLocationTileRanker {
         if (newCityTile.hasViewableResource(civ) && newCityTile.tileResource.resourceType == ResourceType.Bonus) tileValue -= 8
         // Settling on bonus resources tends to waste a food
         // Settling on luxuries generally speeds up our game, and settling on strategics as well, as the AI cheats and can see them.
+        
+        tileValue -= civ.cities.size * 5 
```

Comment:
This will mean that at 20 cities I'll basically be able to settle nowhere, no?

Reply:
Yes, it's a cap on the number of cities.

Reply:
Why would we want to disable AI from settling anywhere after X cities?

Reply:
The threshold for minimum tile score needs to be lower than 50 points, so the AI will settle also with bad spawn or weird ruleset/weird map settings, but them sprawling cities on empty snow/tundra/desert is not something a human would do and it feels 'unnatural' to the user. Setting the threshold low and increasing with more cities is maybe a way to balance these two problems.

Reply:
Yes, but this config would have them not sprawl even on super good tiles, which is also unnatural

Reply:
How do I look up the average value on land tiles on the map? The best way then may be to let the AI calculate this for the whole map, and then use that to set a baseline for minimum tile quality.